crates/sui-framework/packages/sui-framework/sources/accumulator.move

```diff
@@ -17,3 +21,115 @@ fun create(ctx: &TxContext) {
         id: object::sui_accumulator_root_object_id(),
     })
 }
+
+/// Balance object methods
+
+/// The key type to look up a balance object.
+public struct AccumulatorName<phantom T> has copy, drop, store {
+    address: address,
+}
+
+fun accumulator_root_has_balance<K, V: store>(
```

Comment:
As a general style thing, we try to avoid repeating the name of the module in functions. And similarly don't really use the prefix `get`, as it comes from a strange places for OO field accessors that we don't really use

So for example this might just want to be `root_has_balance` or even `has_balance`

Reply:
I wasn't repeating the module name, but the struct name, but I can shorten it either way